### ISSUE TITLE
[proposal]  remove "loading" code, and prepare for incremental map loading.

### ISSUE THREAD

**Issue by benmoran56** [XenonCoder/TerraCraft#31]
I was thinking that we should remove the "now loading" label from the GameScene. Instead of waiting for the level to be generated, we will want to add incremental loading. This will be important when we try to create larger maps in the future; it won't be practical to wait for the entire map to be generated. Also, when we implement a better generation algorithms, it will be better if we don't have to wait.

**Comment by XenonCoder** [comment]
Yes, exactly what I described to you some time ago. This is a very important thing, plus it improves loading and map generation times.

**Comment by benmoran56** [comment]
OK great. I will experiment with some methods to do this.

**Comment by benmoran56** [comment]
I made a small pull request that changes the behavior slightly. It has more "pop in" now, but I think we can fix that later by improving the map generation/loading code.

**Comment by XenonCoder** [comment]
Good work, now the project is much more organized and easy to manage and read. Thanks so much.
